pkg/reconciler/revision/background_test.go

```diff
@@ -212,6 +212,82 @@ func TestRateLimit(t *testing.T) {
 	}
 }
 
+func TestRateLimitPerItem(t *testing.T) {
+	logger := logtesting.TestLogger(t)
+
+	var resolves atomic.Int32
```

Comment:
This doesn't seem used anywhere here. bad :copyright:  & :spaghetti: ?

Reply:
heh oops, yeah good catch! fixed